app/src/main/java/com/madonasyombua/growwithgoogleteamproject/ui/fragment/RegisterFragment.java

```diff
@@ -54,18 +43,6 @@ public void onClick(View view) {
                 String username_txt = extractText(binding.editUsername);
```

Comment:
Please use naming convention , we should use camelCase in our case if the variable is member we name like that mVariable

Reply:
Yeah, like mMaster :) Agreed. So far so good project.